src/Styleguide/Pages/Artist/Routes/Shows/ShowsRefetchContainer.tsx

```diff
@@ -138,8 +138,10 @@ export const ShowsRefetchContainer = createRefetchContainer(
                   <Row>
                     <Col>
                       <Flex justifyContent="flex-end">
-                        <Pagination
-                          {...this.props.artist.showsConnection.pageCursors}
+                        <PaginationFragmentContainer
+                          pageCursors={
+                            this.props.artist.showsConnection.pageCursors as any
+                          }
                           onClick={this.loadAfter}
                           onNext={this.loadNext}
                           onPrev={this.loadPrev}
```

Comment:
👍 i think we can soon consolidate this stuff too.

Reply:
I was trying to (since with the new 'Previous' implementation have to go in and adjust 4 of these `loadPrev` methods), but they all relay on the `relay` prop thats passed in to the container (from the query renderer), and I was having some trouble moving these into their own file (a helper method that accepts a `relay` variable) kinda thing.

Reply:
Ah, i see... yeah i think we can fix this by using `context` as its flowing through app